var/spack/repos/builtin/packages/py-python-htmlgen/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyPythonHtmlgen(PythonPackage):
+    """Library to generate HTML from classes.
+    """
+
+    homepage = "https://github.com/srittau/python-htmlgen"
+    url      = "https://github.com/srittau/python-htmlgen/archive/v1.2.2.tar.gz"
```

Comment:
I think we prefer pypi.io over github urls for python packages. But no strong feelings

Reply:
It doesn't seem like source files are available under pypi.io: https://pypi.org/project/htmlgen/#files So, I will keep the url if that is ok.